code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -1,133 +1,99 @@
-#define UNDERWEAR_LAYER  		5
-#define SHOES_LAYER_ALT			6
-#define UNIFORM_LAYER			7
-#define ID_LAYER				8
-#define SHOES_LAYER				9
-#define GLOVES_LAYER			10
-#define BELT_LAYER				11
-#define SUIT_LAYER				12
-#define TAIL_LAYER				13		//bs12 specific.	//In a perfect world the parts of the tail that show between legs would be on a new layer. Until then, sprite's been tweaked
-#define GLASSES_LAYER			14
-#define BELT_LAYER_ALT			15
-#define SUIT_STORE_LAYER		16
-#define BACK_LAYER				17
-#define HAIR_LAYER				18		//TODO: make part of head layer?
-#define EARS_LAYER				19
-#define EYES_LAYER				20
-#define FACEMASK_LAYER			21
-#define HEAD_LAYER				22
-#define COLLAR_LAYER			23
-#define HANDCUFF_LAYER			24
-#define LEGCUFF_LAYER			25
-#define L_HAND_LAYER			26
-#define R_HAND_LAYER			27
-#define MODIFIER_EFFECTS_LAYER	28
-#define FIRE_LAYER				29		//If you're on fire
-#define WATER_LAYER				30		//If you're submerged in water.
-#define TARGETED_LAYER			31		//BS12: Layer for the target overlay from weapon targeting system
-#define TOTAL_LAYERS			31
+#define MUTATIONS_LAYER			1		//Mutations like fat, and lasereyes
+#define SKIN_LAYER				2		//Skin things added by a call on species
+#define BLOOD_LAYER				3		//Bloodied hands/feet/anything else
+#define DAMAGE_LAYER			4		//Injury overlay sprites like open wounds
+#define SURGERY_LAYER			5		//Overlays for open surgical sites
+#define UNDERWEAR_LAYER  		6		//Underwear/bras/etc
+#define SHOES_LAYER_ALT			7		//Shoe-slot item (when set to be under uniform via verb)
+#define UNIFORM_LAYER			8		//Uniform-slot item
+#define ID_LAYER				9		//ID-slot item
+#define SHOES_LAYER				10		//Shoe-slot item
+#define GLOVES_LAYER			11		//Glove-slot item
+#define BELT_LAYER				12		//Belt-slot item
+#define SUIT_LAYER				13		//Suit-slot item
+#define TAIL_LAYER				14		//Some species have tails to render
+#define GLASSES_LAYER			15		//Eye-slot item
+#define BELT_LAYER_ALT			16		//Belt-slot item (when set to be above suit via verb)
+#define SUIT_STORE_LAYER		17		//Suit storage-slot item
+#define BACK_LAYER				18		//Back-slot item
+#define HAIR_LAYER				19		//The human's hair
+#define EARS_LAYER				20		//Both ear-slot items (combined image)
+#define EYES_LAYER				21		//Mob's eyes (used for glowing eyes)
+#define FACEMASK_LAYER			22		//Mask-slot item
+#define HEAD_LAYER				23		//Head-slot item
+#define HANDCUFF_LAYER			24		//Handcuffs, if the human is handcuffed, in a secret inv slot
+#define LEGCUFF_LAYER			25		//Same as handcuffs, for legcuffs
+#define L_HAND_LAYER			26		//Left-hand item
+#define R_HAND_LAYER			27		//Right-hand item
+#define MODIFIER_EFFECTS_LAYER	28		//Effects drawn by modifiers
+#define FIRE_LAYER				29		//'Mob on fire' overlay layer
+#define WATER_LAYER				30		//'Mob submerged' overlay layer
+#define TARGETED_LAYER			31		//'Aimed at' overlay layer
+#define TOTAL_LAYERS			32//<---- KEEP THIS UPDATED, should always equal the highest number here, used to initialize a list.

```

Comment:
Highest number here, or highest number+1?

Reply:
Should be highest number. Won't hurt anything for being LONGER, only SHORTER though, so this wouldn't actually cause an issue, but I'll fix it now with the other comment above if you're ready to merge.